scheduler/context/context.py

```diff
@@ -35,47 +35,47 @@ def __init__(self) -> None:
             logging.config.dictConfig(json.load(f))
 
         # Services
-        svc_katalogus = services.Katalogus(
+        katalogus = services.Katalogus(
             host=self.config.host_katalogus,
             source=f"scheduler/{scheduler.__version__}",
         )
 
-        svc_bytes = services.Bytes(
+        bytes = services.Bytes(
```

Comment:
Nice stuff! Keep in mind that this shadows the built-in `bytes`. An alternative would be to postfix these variables with `_service`. This won't be a Hungarian notation "violation" because `_service` doesn't refer to the *type* but rather to *what it is*. For example: `mail_service`, `core_controller`, `home_view`, `user_manager`, etc.

Anyways it doesn't really matter too much so I'll leave it up to you!